lms/config/__init__.py

```diff
@@ -25,6 +25,12 @@ def configure(settings):
         "hashed_pw": sg.get("HASHED_PW"),
         "salt": sg.get("SALT"),
         "username": sg.get("USERNAME"),
+        # The secret string that's used to sign the session cookie.
+        # This needs to be a 64-byte, securely generated random string.
+        # For example you can generate one using Python 3 on the command line
+        # like this:
+        #     python3 -c 'import secrets; print(secrets.token_hex(nbytes=64))'
```

Comment:
I didn't know about the secrets stdlib module. That's useful.

Reply:
It's new in Python 3